resources/lang/de/filter.json

```
@@ -0,0 +1,6 @@
+{
+  "Select option": "Auswählen",
+  "Press enter to select": "Enter, um auszuwählen",
+  "Selected": "Ausgewählt",
+  "Press enter to remove": "Enter, um zu entfernen"
```

Maybe a better translation:
```suggestion
  "Press enter to remove": "Enter drücken, um zu entfernen"
```

---

I deliberately kept it short. There is not much space.